app/services/LandingService.scala

```diff
@@ -283,7 +283,11 @@ class LandingService @Inject() (
             (_.transactionsInNext12Months.map(t => TransactionsInLast12Months(t.txnAmount))),
 
           sendTheLargestAmountsOfMoney = viewResponse.msbSection.fold[Option[SendTheLargestAmountsOfMoney]](None)
-            (_.sendTheLargestAmountsOfMoney.map(s => SendTheLargestAmountsOfMoney(s.country_1, s.country_2, s.country_3))),
+            (_.sendTheLargestAmountsOfMoney.map (s => s.countries match {
+              case Seq(a, b, c) => SendTheLargestAmountsOfMoney(a, Some(b), Some(c))
```

Comment:
Could maybe do something with .head and .tail

Reply:
But i'm not massively keen

Reply:
if you don't mind I'll leave this as there are only max three elements in seq

Reply:
Agreed